actors/sunbird-lms-mw/actors/sunbird-utils/sunbird-platform-core/common-util/pom.xml

```diff
@@ -154,7 +154,7 @@
 		<dependency>
 			<groupId>com.googlecode.libphonenumber</groupId>
 			<artifactId>libphonenumber</artifactId>
-			<version>8.10.2</version>
+			<version>8.12.5</version>
```

Comment:
Can you check this library change and its significance?

Reply:
to validate the new phone number, as this library validating the phone number based on area code.